Description: Use versions of build-deps available in Debian
Author: Arnaud Ferraris <aferraris@debian.org>
Forwarded: not-needed
Last-Update: 2024-10-01
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -129,4 +129,4 @@
 unbounded_depth = ["heed-types/unbounded_depth"]
 
 [target."cfg(windows)".dependencies.url]
-version = "2.5.2"
+version = "2.5"
